### DISCUSSION TITLE
"Modeception"

### DISCUSSION THREAD

**Discussion by SiggiStonk** [bwRavencl/ControllerBuddy#82] [Ideas]
I tried to set up ControllerBuddy for Star Citizen and couldn't quite figure it out. 
Is it somehow possible to bind a mode switch action inside another mode than the default one?
E.g.: Default Mode -> press 'B' -> Mode 1 -> press 'B' -> Mode 2 -> press 'B' -> Default again.

It would also be possible to set one button as the default mode in every submode.

Hope my point is understandable. :)

**Comment by bwRavencl** [comment]
Unfortunately what you are describing is not supported by ControllerBuddy at this point and it is also not a feature I can quickly implement.
To allow binding Mode Switching Actions inside other Modes than the default Mode would require some pretty extensive architectural changes and lots of testing to make it bullet-proof.
This is something I might consider implementing on day, but you should really not hold your breath for it.
